maze_transformer/training/dataset.py

```diff
@@ -112,6 +112,7 @@ def to_fname(self) -> str:
         return sanitize_fname(f"f{self.name}_{self_json_hash}")
 
 
+# TODO: This class is not referenced anywhere. Can we delete?
```

Comment:
yeah, this class should be deleted. I might add it to muutils though since it seems generally useful.